pilot.go

```diff
@@ -260,16 +260,35 @@ func initAutoPilot(svr *server, cfg *lncfg.AutoPilot,
 			chanState := make([]autopilot.LocalChannel,
 				len(activeChannels))
 			for i, channel := range activeChannels {
+				localCommit := channel.LocalCommitment
+				balance := localCommit.LocalBalance.ToSatoshis()
+
 				chanState[i] = autopilot.LocalChannel{
-					ChanID:   channel.ShortChanID(),
-					Capacity: channel.Capacity,
+					ChanID:  channel.ShortChanID(),
+					Balance: balance,
 					Node: autopilot.NewNodeID(
 						channel.IdentityPub),
 				}
 			}
 
 			return chanState, nil
 		},
+		ChannelInfo: func(chanPoint wire.OutPoint) (
+			autopilot.LocalChannel, error) {
```

Comment:
why not return `*autopilot.LocalChannel`, that way returning an error would be cleaner (`nil` return value)?

Reply:
It was just to not have to dereference the returned value before passing it to `OnChannelEdge`, but changed now: https://github.com/lightningnetwork/lnd/pull/4666/commits/2a3114641e27db7d640a0d8dd09ccb3cdc974710